include/openssl/aes.h

```diff
@@ -7,8 +7,11 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef HEADER_AES_H
-# define HEADER_AES_H
+#ifndef OPENSSL_AES_H
+# define OPENSSL_AES_H
+# pragma once
+
+# define HEADER_AES_H /* deprecated in version 3.0 */
```

Comment:
Why do we talk about 3.0 deprecation this in 1.1.1?

Reply:
I'll remove the comment.